includes/prompt.h

```diff
@@ -0,0 +1,28 @@
+#ifndef PROMPT_H
+# define PROMPT_H
+
+#include "libft.h"
+#include <stdlib.h>
+
+#define BUFFER_SIZE 255
```

Comment:
this is a perfect example of collision. You have defined BUFFER_SIZE in directory.h as well. never do this

Reply:
also, does the library that includes prompt.h care about the buffer size? if the answer is no, don't put it in the header. put it in the C file